src/foremast/pipeline/construct_pipeline_block.py

```diff
@@ -142,6 +142,12 @@ def construct_pipeline_block(env='',
     hc_grace_period = data['asg'].get('hc_grace_period')
     app_grace_period = data['asg'].get('app_grace_period')
     grace_period = hc_grace_period + app_grace_period
+    
+    ssh_keypair = data['asg'].get('ssh_keypair', None)
+    LOG.info('SSH keypair ({0}) used'.format(ssh_keypair))
+    if not ssh_keypair:
+        ssh_keypair = '{0}_{1}_default'.format(env, region)
```

Comment:
Added a TODO to migrate this out but explained the issue that if we specify an temp value, region is not available when configs.json.j2 is parsed just yet, we talked about externalizing this bit to gogo-utils. Added a todo